src/lib/lightning.ts

```diff
@@ -9,14 +9,18 @@ import {
 } from '@arkade-os/boltz-swap'
 import { RestArkProvider, RestIndexerProvider } from '@arkade-os/sdk'
 import { AspInfo } from '../providers/asp'
+import { BoltzUrl } from './constants'
 
 export class LightningSwap {
   private readonly provider: ArkadeLightning
   private readonly swapProvider: BoltzSwapProvider
 
   constructor(aspInfo: AspInfo, wallet: Wallet) {
     const arkProvider = new RestArkProvider(aspInfo.url)
-    const swapProvider = new BoltzSwapProvider({ network: aspInfo.network as Network })
+    const swapProvider = new BoltzSwapProvider({
+      network: aspInfo.network as Network,
+      apiUrl: BoltzUrl(),
```

Comment:
No need to be a function

```suggestion
      apiUrl: BoltzUrl,
```